pkg/minikube/service/service.go

```diff
@@ -156,7 +127,8 @@ func GetServiceURLs(api libmachine.API, namespace string, t *template.Template)
 
 // GetServiceURLsForService returns a SvcUrl object for a service in a namespace. Supports optional formatting.
 func GetServiceURLsForService(api libmachine.API, namespace, service string, t *template.Template) (SvcURL, error) {
```

Comment:
would it be out of scope to pass the profile name instead of viper ?

Reply:
Done.